typings/internalBinding/os.d.ts

```diff
@@ -0,0 +1,21 @@
+declare function InternalBinding(binding: 'os'): {
+  getHostname(ctx: object): string | undefined;
```

Comment:
Maybe we can use `{}` instead for `ctx`?
https://github.com/nodejs/node/blob/152d675cc6a53dc5a3528049216dd52d512084a7/lib/os.js#L65